src/data/npcs.asm

```diff
@@ -113,411 +114,380 @@ PointerTable_118f5:
 	dw Data_11f42
 	dw Data_11f49
 	dw Data_11f49
-	dw Data_11c33
-	dw Data_119f7
-	dw Data_11a04
+	dw Murray2Data
+	dw Ronald2Data
+	dw Ronald3Data
 	dw Data_11f49
-Data_119dd:
-	db $01
+DrMasonData:
+	db DRMASON
 	db $02
 	db $00
 	db $26
 	db $00
-	dw $5727 ; Pointer to NPC Data
-	db $ac
-	db $03
+	dw $5727 ; Pointer to OWScript
+	tx Text03ac
 	db $00
 	db $00
 	db $00
 	db $00
-Data_119ea:
-	db $02
+Ronald1Data:
+	db RONALD1
 	db $01
 	db $04
 	db $0e
 	db $00
-	dw $5c4b ; Pointer to NPC Data
-	db $ad
-	db $03
-	db $02
-	db $1a
+	dw OWSequence_Ronald ; Pointer to OWScript
```

Comment:
I would suggest using a `Script_` prefix or `Script` suffix instead of `OWSequence_`. It feels a little too verbose.
We can compare with my label choices in #67

Reply:
the reason I went with OWSequence is because I named every script function with an OWScript prefix. I'm not sure what I'd change it to if not OWScript. Maybe ScriptFunction_? That's a bit verbose too though